cdap-app-fabric/src/main/java/io/cdap/cdap/internal/capability/CapabilityNotAvailableException.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.internal.capability;
+
+/**
+ * CapabilityNotAvailableException indicating that the capability is disabled
+ */
+public class CapabilityNotAvailableException extends Exception {
```

Comment:
Have this class also implements the `HttpErrorStatusProvider` interface to respond with appropriate HTTP status when a capability is not available during app deploy / program start time.

Reply:
Done